### PR TITLE
Usertimeline backend

### PR DESCRIPTION
**Pull request by Resident-png**
For now, a user can see their own posts from the newest to the latest. Everytime you make a post it refreshes the page to show you your latest post. Only problem with the current implementation is that the usertimeline does not load the posts the first time you log onto your account, so you need to refresh the page the first time you get on.

Other important note: I was able to get the user connections without any problem (that I or vscode could detect), but when I try to get those connections posts (and display them) it does not work. I have commented out that part of the code which does not work. I'll see if I'm able to get this to work before the end of tonight.

### CODE REVIEW

File: src/client/src/components/UserTimeline.jsx [hunk 1]
```diff
@@ -69,10 +80,57 @@ const UserTimeline = () => {
...
+
+  const allConnectionsPosts = connectionsPosts.map((connectionsPost) => (
+    <div className="userPostsTimeline">
+      <span className="subTitleTimeline">{connectionsPost.firstname}{" "}{connectionsPost.lastname}</span>
+      <p className="postText">
+        {connectionsPost.description}
+      </p>
+    </div>
+  ))*/
+
```
Comment (imran1289-ah) [inline src/client/src/components/UserTimeline.jsx:112]: remove uncommented things if not using it

**Review by imran1289-ah — APPROVED** [review]
everything seems good. look comment

**Comment by imran1289-ah** [comment]
there is a bug occuring with this user story #122

**Comment by Resident-png** [comment]
since no one seems to have any further comments, i will merge this pull request with dev_new

**Pull request merged**